index.html

```diff
@@ -418,9 +418,11 @@ <h2>JOSE Header Parameters and JWT Claims</h2>
         secured and semantics provided by the [[VC-DATA-MODEL-2.0]]. This means that while the security
         features of this specification ensure data integrity and authenticity, they do
         not dictate the interpretation of claim data.
-        
-        Implementers are RECOMMENDED to avoid conflicting values, especially with claims such as
-        `issuer` with `iss`, `id` with `jti`, and `credentialSubject.id` with `sub`.
+      </p>
+      <p>
+        Implementers SHOULD avoid setting JWT claims to values that conflict with
+        verifiable credential properties, especially with pairs such as
```

Comment:
verifiable credential properties when the claims and properties refer to the same concept, for example,